x/emissions/migrations/v6/migrate.go

```diff
@@ -106,9 +106,11 @@ func MigrateParams(store storetypes.KVStore, cdc codec.BinaryCodec) error {
 		InitialRegretQuantile:               oldParams.InitialRegretQuantile,
 		PNormSafeDiv:                        oldParams.PNormSafeDiv,
 		// NEW PARAMS
-		GlobalWhitelistEnabled:       defaultParams.GlobalWhitelistEnabled,
-		TopicCreatorWhitelistEnabled: defaultParams.TopicCreatorWhitelistEnabled,
-		MinExperiencedWorkerRegrets:  defaultParams.MinExperiencedWorkerRegrets,
+		GlobalWhitelistEnabled:             defaultParams.GlobalWhitelistEnabled,
+		TopicCreatorWhitelistEnabled:       defaultParams.TopicCreatorWhitelistEnabled,
+		MinExperiencedWorkerRegrets:        defaultParams.MinExperiencedWorkerRegrets,
+		InferenceOutlierDetectionThreshold: defaultParams.InferenceOutlierDetectionThreshold,
+		InferenceOutlierDetectionAlpha:     defaultParams.InferenceOutlierDetectionAlpha,
```

Comment:
No new migration needed because we could simply use the default values and/or set the values in a tx after the upgrade, right?

Reply:
That was my understanding about the params from previous migrations - does that also apply to the new stores defined in keeper? (lastMedian, madEma)

Reply:
If a network is already running a 'v0.7.0' version these default values won't be set.

The additions in the stores are consensus breaking, a node with a previous version must be updated so we should increment the consensus version and add a migration to set default values for new params.

Regarding the proto API I think we can keep the `v6` as the addition of fields is not breaking, marshalling and unmarshalling of the `Params` struct will be backward and forward compatible with the addition of these parameters.

Reply:
@xmariachi We can omit setting the default values in a migration and set them with an `UpdateParams` tx after, but having new stores in the state is consensus breaking as it changes the computed app hash for the block, this required a coordinated upgrade and that suggests a `v0.8.0` I guess

Reply:
Doing `v0.8.0` migration

Reply:
Pushed `v0.8.0` migration tested locally.